### ISSUE TITLE
BUG: --extract-ipv4s does not work

### ISSUE THREAD

**Issue by ZeroDot1** [InQuest/iocextract#73]
Unfortunately it doesn't work, I ran it for quite a while but except for stressing one CPU core 100% nothing happened, the IPs were not written to the file.
`iocextract --input '/home/user/des.txt' --output '/home/user/k1.txt' --extract-ipv4s`

**Comment by azazelm3dj3d** [comment]
Hi, @ZeroDot1

I just ran a quick assessment and had no issues extracting the IP addresses from the test data.

Here's my dataset ("`des.txt`"):

```txt
asdf192[.]168[.]0[.]1:80/pathtttttt192.168.0.1ssssss192[.]168[.]0[.]1loremipsum
```

My command:

```bash
iocextract --input 'des.txt' --output 'k1.txt' --extract-ipv4s
```

If possible, could you provide the data you're attempting to extract from? Or provide an example?

**Comment by azazelm3dj3d** [comment]
That may be the issue, but I've definitely tested up to 20mb-30mb in the past. I was able to extract IOCs from the test file using the command I mentioned in the previous response without any issues and it took less than a second to produce.

I also just tested a 20mb file from SecLists full of randomized characters and only 2 IP addresses throughout the entire file. It was able to find the IP addresses within a few seconds. Here are the results:

```bash
Command: iocextract --input 'des.txt' --output 'k1.txt' --extract-ipv4s 
User: 1.64s
System: 0.03s
CPU: 99%
Total: 1.678s
```

I'm going to close this issue as complete for now, but I will definitely look into testing larger datasets (~50mb+) when I have the time. Hopefully, you're able to resolve the issue and everything works out soon. My one recommendation is to try to cut the file size down if possible and see if that improves the results.
